packages/harvester-csi-driver/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/harvester/charts/releases/download/harvester-csi-driver-0.1.4/harvester-csi-driver-0.1.4.tgz
+packageVersion: 00
+releaseCandidateVersion: 00
```

Comment:
dev-v2.6 does not utilize `packageVersion` or `releaseCandidateVersion`

Can you switch this to `version: 100.0.0`?